common/__init__.py

```diff
@@ -1,9 +1,14 @@
 import os
 
 from qdrant_client import QdrantClient
-from global_utils import CloudFunctionHandler, LocalFunctionHandler
 
-env_handler = LocalFunctionHandler() if os.getenv("TYPE") == "LOCAL" else CloudFunctionHandler()
+from common.handler_cloud import CloudFunctionHandler
+from common.handler_local import LocalFunctionHandler
+
+
+env_handler = (
+    LocalFunctionHandler() if os.getenv("TYPE") == "LOCAL" else CloudFunctionHandler()
```

Comment:
Its okay, keep in mind that you can create dict 
```
{
"LOCAL": LocalFunctionHandler,
"CLOUD": CloudFunctionHandler,
}
```
which can be easier expanded with new features and also you can raise custom error on key exception,
but you dont need to change that, its just a suggestion